repository-data/application/src/main/resources/hcm-config/namespaces/website/roadmapItem.yaml

```diff
@@ -139,6 +137,14 @@ definitions:
         jcr:primaryType: editor:templateset
       /hipposysedit:nodetype:
         /hipposysedit:nodetype:
+          /categorylink:
+            hipposysedit:mandatory: false
+            hipposysedit:multiple: false
```

Comment:
Should this be multiple to allow multiple categories to be assigned a roadmap item or is it just 1 category per roadmap item now?

Reply:
1 for now